blueprints/third-party-solutions/phpipam/variables.tf

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Documentation: https://cloud.google.com/run/docs/securing/managing-access#making_a_service_public
+
+variable "admin_principals" {
+  description = "Map of identities to give rights to (CloudSQL admin, client and instanceUser, Logging admin, Service Account User and TokenCreator)."
+  type = object({
+    groups           = optional(list(string), [])
+    service_accounts = optional(list(string), [])
+    users            = optional(list(string), [])
+  })
+  default = {
+    groups           = []
+    service_accounts = []
+    users            = []
+  }
```

Comment:
1. I'm still not convinced about the type. Why not just a single list(string) and let the user build the identity (i.e. let them add the user/group/sa preffix)?
2. In any case `default={}` should be enough here